### PR TITLE
Toolbar links functionality and styles

### PR DESCRIPTION
**Pull request by alanshaw**
screen shot 2017-10-20 at 15 58 07

### CODE REVIEW

**Comment by olizilla** [comment]
👁 👁

File: src/components/dropdown/Dropleft.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    display: open ? null : 'none',
+    position: 'absolute',
+    top: '50%',
+    left
+  }} data-id='dropleft-menu'>
+    {children}
+  </div>
+)
+
+const MenuArrowRight = ({height = 12, align = 'middle', marginTop = 'auto', marginBottom = 'auto'}) => {
```
Comment (olizilla) [inline src/components/dropdown/Dropleft.js]: You are open-sauce-ing my secret-sauce! It's been my family's secret arrow recipe for generation!

(ノಠ益ಠ)ノ彡┻━┻

File: src/components/toolbar/buttons/LinkButton.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import Button from './Button'
+import { LinkIcon } from '../../icons'
+import { Dropleft, DropleftMenu } from '../../dropdown/Dropleft'
+
+export default class LinkButton extends Component {
```
Comment (olizilla) [inline src/components/toolbar/buttons/LinkButton.js]: Took me a while to figure out what `LinkButton` was... perhaps `PadUrlsButton`

File: src/components/toolbar/buttons/Button.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Button = ({ theme, icon: Icon, title, onClick }) => (
```
Comment (olizilla) [inline src/components/toolbar/buttons/Button.js]: `IconButton` ?

**Comment by alanshaw** [comment]
I'll send the renames as a separate PR

File: src/components/dropdown/Dropleft.js (hunk 1 above)
Comment (alanshaw) [reply src/components/dropdown/Dropleft.js]: It is the good sauce!

**Pull request merged**